### DISCUSSION TITLE
How to setup the Template Document in .doc instead .pdf format

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14772] [How to]
Hello,
I would to know if it is possible to setup the template for the document in .doc format instead .pdf

Regards

**Comment by mariuszkrzaczkowski** [comment]
currently this is not possible

**Comment by apcloic** [comment]
Maybe you could add phpword dependencies
https://github.com/PHPOffice/PHPWord
And then write your own functions.
Regards

**Reply by espe808** [reply]
@mariuszkrzaczkowski https://github.com/PHPOffice/PHPWord
Could be a solution?

**Reply by mariuszkrzaczkowski** [reply]
I mean crm doesn't have ready mechanisms, there are possibilities, only someone has to add it to the system

**Reply by espe808** [reply]
I will try to do in that way 🙂
